Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -713,6 +716,12 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
         // all at the same time by too many users concurrently.
         self.updateCachesIfInForeground()
 
+        #if DEBUG
```

Comment:
I believe we should also check this
```suggestion
        #if DEBUG && !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
```
This way, we don't really care about the value of `validateConfigurationOnLaunch` when `ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION`